session.go

```diff
@@ -219,10 +220,15 @@ func (s *schemaLeaseChecker) Check(txnTS uint64) error {
 func (s *schemaLeaseChecker) checkOnce(txnTS uint64) error {
 	succ := s.SchemaValidator.Check(txnTS, s.schemaVer)
```

Comment:
After accelerate ddl in RC3, some contract does not hold any more, I think `SchemaValidator` have to be redesigned.